civihr_default_theme/assets/sass/civihr_default/_custom.scss

```diff
@@ -1045,3 +1045,21 @@ form {
 .jcarousel-container [disabled] {
     display: none;
 }
+
+/* Styling the exposed filters on reports page. */
+@media (min-width: 768px) { 
+    #edit-absence-date-filter-wrapper, #edit-between-date-filter-wrapper {
+        .form-inline {
+            .form-control {
+                display: inline-block;
+                width: auto;
+                vertical-align: middle;
+            }
+            .form-group {
+                display: inline-block;
+                margin-bottom: 0;
+                vertical-align: middle;
+            }
+        }
+    }  
+}  
```

Comment:
1) We're switching gradually our code base to a 2-space indentation, please make sure to use that in your changes
2) Always leave a new line at the of of the file

We have a [.editorconfig](https://github.com/compucorp/civihr-employee-portal-theme/blob/develop/.editorconfig) already in our repo that should help ensure these (and other) rules via your IDE, please make sure you configure it correctly (see [here](http://editorconfig.org/#download))